package.json

```diff
@@ -59,6 +59,13 @@
     "build:dist": "webpack --config conf/webpack.config.js --sort-modules-by size && mkdir -p examples/browser/lib && cp dist/orbitdb.min.js examples/browser/lib/orbitdb.min.js",
     "build:debug": "webpack --config conf/webpack.debug.config.js --sort-modules-by size && mkdir -p examples/browser/lib && cp dist/orbitdb.js examples/browser/lib/orbitdb.js && cp dist/orbitdb.js.map examples/browser/lib/orbitdb.js.map",
     "build:docs/toc": "markdown-toc --no-first1 -i README.md && markdown-toc --no-first1 -i API.md && markdown-toc --no-first1 -i GUIDE.md && markdown-toc --no-first1 -i CHANGELOG.md",
-    "build:es5": "babel src --out-dir ./dist/es5/ --presets babel-preset-es2015 --plugins babel-plugin-transform-runtime"
+    "build:es5": "babel src --out-dir ./dist/es5/ --presets babel-preset-es2015 --plugins babel-plugin-transform-runtime",
+    "test-win": "node node_modules\\mocha\\bin\\mocha",
+    "build-win": "npm run build:es5-win && npm run build:debug-win && npm run build:dist-win && npm run build:examples-win && npm run build:docs/toc-win",
+    "build:examples-win": "webpack --config conf/webpack.example.config.js --sort-modules-by size && if not exist examples\\browser\\lib mkdir examples\\browser\\lib && cp node_modules\\ipfs\\dist\\index.js examples\\browser\\lib\\ipfs.js",
+    "build:dist-win": "webpack --config conf\\webpack.config.js --sort-modules-by size && if not exist examples\\browser\\lib mkdir examples\\browser\\lib && cp dist\\orbitdb.min.js examples\\browser\\lib\\orbitdb.min.js",
+    "build:debug-win": "webpack --config conf\\webpack.debug.config.js --sort-modules-by size && if not exist examples\\browser\\lib mkdir examples\\browser\\lib && cp dist\\orbitdb.js examples\\browser\\lib\\orbitdb.js && cp dist\\orbitdb.js.map examples\\browser\\lib\\orbitdb.js.map",
+    "build:docs/toc-win": "markdown-toc --no-first1 -i README.md && markdown-toc --no-first1 -i API.md && markdown-toc --no-first1 -i GUIDE.md && markdown-toc --no-first1 -i CHANGELOG.md",
```

Comment:
This line seems to be the same as https://github.com/orbitdb/orbit-db/pull/453/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R61. Do we need a separate script for Windows or could we use the same scripts as above?

Reply:
You are right, this line isn't really necessary. I'll delete it.